main.go

```diff
@@ -168,6 +173,23 @@ func isBitriseCacheAPIURL(url string) bool {
 	return url == os.Getenv("BITRISE_CACHE_API_URL")
 }
 
+func writeCachePullTimestamp() error {
+	f, err := os.Create(cachePullEndTimePath)
+
+	if err != nil {
+		return err
+	}
+
+	defer f.Close()
```

Comment:
missing error handling, `errcheck` step should fail because of it.

usually, we just print a warning message if an error occurs in a deferred file close: https://github.com/bitrise-steplib/steps-cache-push/blob/master/cache_archive.go#L102-L106